packages/artifact/src/__mocks__/internal-config-variables.ts

```diff
@@ -12,6 +12,14 @@ export function getUploadChunkConcurrency(): number {
 export function getUploadChunkSize(): number {
   return 4 * 1024 * 1024 // 4 MB Chunks
 }
+
+export function getUploadRetryCount(): number {
+  return 1
+}
+
+export function getRetryWaitTime(): number {
+  return 1
```

Comment:
Would be clearer if we put the unit (milliseconds, second, or minute) in the function, `getRetryWaitTimeInSeconds()` or something like that.